File: C/23.c
#include<stdio.h>
void main()
{
    int counter,size,array[50],searchnum;
    printf("Enter the size of array : ");
    scanf("%d",&size);
    printf("Enter array elements : \n");
    for(counter=0;counter<size;counter++)
    {
        printf("Element %d : ",(counter+1));
        scanf("%d",&array[counter]);
    }
    printf("Enter number to search : ");
    scanf("%d",&searchnum);
    for(counter=0;counter<size;counter++)
    {
        if(array[counter]==searchnum)
        break;
    }
    if(counter==size)
    printf("Element not found.");
    else
    printf("Element found at position %d.",(counter+1));
    printf("\n");
}

File: Python/23.py
a = []
n = int(input('Enter the number of elements: '))
print('Enter elements:')
for i in range(n):
    a.append(int(input()))

print('Odd numbers: ',end='')
for i in range(len(a)):
    if a[i] % 2 != 0:
        print(a[i],end=' ')